ui/components/src/LineChart/LineChart.tsx

```diff
@@ -20,6 +21,7 @@ import type {
   LineSeriesOption,
   LegendComponentOption,
   VisualMapComponentOption,
+  YAXisComponentOption,
```

Comment:
This is a typo in one of the types that echarts exports, I'm probably going to submit a PR in their repo to fix it